src/Sankey.js

```diff
@@ -0,0 +1,201 @@
+import {sankey, sankeyLinkHorizontal} from "d3-sankey";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Path, Rect} from "d3plus-shape";
+import {dataLoad as load, Viz} from "d3plus-viz";
+
+/**
+    @class Rings
+    @extends external:Viz
```

Comment:
in orer for the external extends link to work in the documentation, you need to include a piece of boilerplate at the top of the file (look how Network does it):

```js
/**
    @external Viz
    @see https://github.com/d3plus/d3plus-viz#Viz
*/
```